UnityProject/Assets/Scripts/Managers/GameManager.cs

```diff
@@ -692,11 +692,17 @@ public void EndRound()
 		}
 
 		counting = false;
-		if (RoundEndTime > 10)
+		try
```

Comment:
so why was this failing? I imagine it was an NRE.

Reply:
yeah, it was because the person who triggered the vote was no one so it would error when trying to say you can't trigger a vote